enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -351,295 +344,180 @@ web.xml or a partial web.xml:
-
-The container provides certain services based on the life cycle state of
-the Service Implementation Bean. Access to services provided by a web
-container in a Java EE environment (e.g. transactions, JNDI access to
-the component’s environment, etc.) must follow the requirements defined
-by the Servlet and Java EE specifications. Access to a
-ServletEndpointContext must meet the requirements defined by the JAX-RPC
-specification section 10.1.3.
-
-====== Web container programming model for JAX-WS
-
-JAX-WS Service Endpoint that run within the web container must follow
-the requirements repeated here.
-
-A JAX-WS Service Endpoint can be single or multi-threaded. A JAX-WS
-Service Endpoint must implement javax.servlet.SingleThreadModel if
-single threaded access is required by the component. A container must
-serialize method requests for a Service Implementation Bean that
-implements the SingleThreadModel interface. Note, the SingleThreadModel
-interface has been deprecated in the Servlet 2.4 specification.
+A Jakarta XML Web Services Service Endpoint can be single or multi-threaded. 
+A Jakarta XML Web Services Service Endpoint must implement 
+jakarta.servlet.SingleThreadModel if single threaded access is required by 
+the component. A container must serialize method requests for a Service 
+Implementation Bean that implements the SingleThreadModel interface. 
+Note, the SingleThreadModel interface has been deprecated in the Servlet 
+2.4 specification.
 
 The Service Implementation Bean must follow these requirements:
 
 * The Service Implementation Bean class must be annotated with either
-javax.jws.WebService or javax.xml.ws.WebServiceProvider annotation. See
-section 5.3.2.1 and 5.3.2.2
-* For developers starting from Java using JAX-WS, javax.jws.WebService
-annotation on Service Implementation Bean may optionally reference an
-SEI but is not required to do so. If SEI is not specified, the Service
-Implementation Bean class implicitly defines a SEI as required by
-section 3.3 of JAX-WS specification. The business methods of the bean
-must be public and must not be final or static. Only those methods that
-are annotated with @WebMethod in the Service Implementation Bean, are
-exposed to the client.
-* For developers starting from WSDL using JAX-WS, the SEI generated from
-the WSDL must be annotated with javax.jws.WebService annotation. Refer
-to section 2.2 of JAX-WS specification. The Service Implementation Bean
-must be annotated with javax.jws.WebService annotation and the
-endpointInterface member attribute must refer to this generated SEI.
-Service Implementation Bean may implement the Service Endpoint
-Interface, but it is not required to do so. The bean must implement all
-the method signatures of the SEI. The business methods of the bean must
-be public and must not be final or static. It may implement other
-methods in addition to those defined by the SEI.
+jakarta.jws.WebService or jakarta.xml.ws.WebServiceProvider annotation. 
```

Comment:
there reference "See section 5.3.2.1 and 5.3.2.2." should be kept (and appropriate links be created)